web/template/team-work-in-progress.gotmpl

```diff
@@ -97,51 +106,196 @@
     </div>
   </div>
 
-  {{ template "pagination" .Pagination }}
-
-  <hr class="govuk-section-break govuk-section-break--s govuk-section-break--visible govuk-!-margin-top-5">
-
-  <table class="govuk-table">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Donor</th>
-        <th scope="col" class="govuk-table__header">Case</th>
-        <th scope="col" class="govuk-table__header">LPA type</th>
-        <th scope="col" class="govuk-table__header">Received</th>
-        <th scope="col" class="govuk-table__header">Allocation</th>
-        <th scope="col" class="govuk-table__header">Status</th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-      {{ range .Cases }}
-        <tr class="govuk-table__row">
-          <th scope="row" class="govuk-table__header">{{ .Donor.DisplayName }}</th>
-          <td class="govuk-table__cell">
-            <a href="{{ sirius (printf "/lpa/person/%d/%d" .Donor.ID .ID) }}" class="govuk-link">
-              {{ .Uid }}
-            </a>
-          </td>
-          <td class="govuk-table__cell">
-            {{ upper .SubType }}
-          </td>
-          <td class="govuk-table__cell">
-            {{ formatDate .ReceiptDate }}
-          </td>
-          <td class="govuk-table__cell">
-            <strong>{{ .Assignee.DisplayName }}</strong>
-          </td>
-          <td class="govuk-table__cell">
-            <strong class="govuk-tag govuk-tag--{{ statusColour .Status }}">
-              {{ .Status }}
-            </strong>
-          </td>
-        </tr>
-      {{ else }}
-        <tr>
-          <td colspan="6">There are currently no cases assigned to the members of {{ .TeamName }}</td>
-        </tr>
-      {{ end }}
-    </tbody>
-  </table>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <button class="govuk-button govuk-button--secondary" data-filter-toggle>
```

Comment:
This could do with `aria-controls`:
```suggestion
      <button class="govuk-button govuk-button--secondary" data-filter-toggle aria-controls="app-filters">
```